charts/posthog/values.yaml

```diff
@@ -48,6 +48,13 @@ events:
     # -- Max pods for the events stack HorizontalPodAutoscaler.
     maxpods: 10
 
+  securityContext:
+    enabled: false
+    runAsUser: 101
```

Comment:
As discussed above, we should have a description for each field

Reply:
Done